javatests/arcs/core/storage/CapabilitiesResolverTest.kt

```diff
@@ -128,21 +128,28 @@ class CapabilitiesResolverTest {
             resolver1.createStorageKey(Capabilities.TiedToArc, thingSchema, handleId),
             VolatileStorageKey::class.java
         )
-        verifyStorageKey(
-            resolver1.createStorageKey(Capabilities.TiedToRuntime, thingSchema, handleId),
-            RamDiskStorageKey::class.java
-        )
-        verifyStorageKey(
-            resolver1.createStorageKey(Capabilities.Persistent, thingSchema, handleId),
-            DatabaseStorageKey::class.java
-        )
+        val ramdiskKey =
+            resolver1.createStorageKey(Capabilities.TiedToRuntime, thingSchema, handleId)
+        verifyStorageKey(ramdiskKey, RamDiskStorageKey::class.java)
+        assertThat(ramdiskKey).isEqualTo(resolver1.createStorageKey(
+            Capabilities.TiedToRuntime,
+            thingSchema,
+            handleId
+        ))
```

Comment:
is this required? it's calling the same method on the same object, with the same args as on line 131, so the result should be the same anyway?

Reply:
If you want to keep it, that's :+1: , but the styling should look like this:

```kotlin
        assertThat(ramdiskKey).isEqualTo(
            resolver1.createStorageKey(
                Capabilities.TiedToRuntime,
                thingSchema,
                handleId
            )
        )
```

Reply:
dunno, this isn't the brightest test ever, but it makes sure that random unique id generation not going on in CapabilitiesResolver...